recipes/doit/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "doit" %}
+{% set version = "0.30.3" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "2988c8450fb3fb281877eb9a16860e1a8873c6ee0bb75ef0d78bb03a18d59ab9" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  entry_points:
+    - doit = doit.__main__:main
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python >=3
```

Comment:
Instead of restricting `python` here, use the `skip` entry above in `build`:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#skipping-builds

````
build:
   ...
   skip: True  # [py2k]
```